cmake/common/px4_base.cmake

```diff
@@ -1039,14 +1039,30 @@ function(px4_generate_parameters_xml)
 		list(APPEND module_list ${PX4_SOURCE_DIR}/src/${module})
 	endforeach()
 
-	add_custom_command(OUTPUT ${OUT}
-		COMMAND ${PYTHON_EXECUTABLE} ${PX4_SOURCE_DIR}/Tools/px_process_params.py
-			-s ${module_list} ${EXTERNAL_MODULES_LOCATION}
-			--board CONFIG_ARCH_${BOARD} --xml --inject-xml
-			--overrides ${OVERRIDES}
-		DEPENDS ${param_src_files} ${PX4_SOURCE_DIR}/Tools/px_process_params.py
-			${PX4_SOURCE_DIR}/Tools/px_generate_params.py
+	if("${BOARD}" STREQUAL "eagle" OR "${BOARD}" STREQUAL "excelsior")
```

Comment:
Instead of specifying the board here, please set a variable like `DISABLE_PARAMS_MODULE_SCOPING` in the eagle & excelsior board cmake configs and use that here.